template-deliverables.md

```diff
@@ -0,0 +1,145 @@
+    <td>The raw template is the basic template for the type of document you are creating. It provides a rough outline of the suggested content and a few embedded writing tips for how to fill in the different sections of the template.</td>
+    <td>Yes, required</td>
+  </tr>
+  <tr>
+    <td>Template guide</td>
+    <td>This guide provides a deeper explanation of how to fill in the raw template. It provides a lightweight introduction to the purpose of this documentation and explains how to fill in each section of the document. Additional content may be included as needed.</td>
+    <td>Yes, required</td>
+  </tr>
+  <tr>
+    <td>Before you start guide</td>
+    <td>This optional guide is designed to help template users understand some of the key research, brainstorming, or pre-writing steps that they might need to do before adding this type of document to a guide. If this documentation type has certain key decisions that need to be made with stakeholders or if it has a dependency on some other type of document or process, that should be mentioned in this guide.</td>
+    <td>No, optional</td>
+  </tr>
+  <tr>
+    <td>Template example</td>
+    <td>The Good Docs Project is in the process of designing a fake documentation project that can provide examples of our templates in action. Until this project is ready, it is optional to provide an example of the template. However, if you want to create a fake example as part of your template set, you may do so.</td>
+    <td>No, optional</td>
+  </tr>
+</table>
+
+Each template deliverable is explained in the following sections.
+
+## Raw template
+
+Example filename: abc-template.md
+
+The raw template is the basic template for the type of document you are creating.
+It provides a rough outline of the suggested content and a few embedded writing tips for how to fill in the different sections of the template.
+The template user will copy this template and begin filling it in or editing it as their starting point in the writing process.
+
+### Content and formatting guidelines
+
+Each raw template should have this content:
+- **The template title** - The type of document this template is for. For example, "Tutorial template" or "README template."" This title should be formatted with the heading 1 weight.
+- **Introductory comment** - Include an embedded writing tip at the start that tells users they should first read the accompanying template guide and before-you-start guide before they fill in the template. Put introductory comments in {curly brackets}.
+- **The template sections** - Each section of the temple should begin with a heading with the heading 2 weight. The recommended content and some embedded writing tips fall under the headings.
+- **Embedded writing tips** - You can provide some lightweight writing tips that provide some tips or hints about what kind of content the template user might choose to put in a section of the document. Put embedded writing tips in {curly brackets}.

```

Comment:
Can we give some examples for these embedded writing tips?

Reply:
The most common would be after specifying the name of a section, have a writing tip that explains what kind of info ought to go in that section. For example:
"
Request Parameters
{Writing tip: This is where you should list the inputs that end users can provide in the request payload. The simplest way to do this is with a table.}
"

Reply:
Thanks for providing the example, Aidan.

One of the open action items is to create a template style guide that will include guidance about how to format embedded writing tips.